admin/authentication/nca-tools-authentication-strategy.js

```diff
@@ -42,10 +43,13 @@ module.exports = async function (req, done) {
     }
 
     const mtcUser = await ncaToolsUserService.mapNcaUserToMtcUser(userData)
+    const school = await schoolDataService.sqlFindOneById(mtcUser.school_id)
```

Comment:
mtcUser.school_id will only be present if the user has a school. Some roles don't.